src/base/object_representation.h

```diff
@@ -200,10 +200,20 @@ struct log_lsa;
   (*(short *) ((char *) (ptr)) = htons ((short) (val)))
 #define OR_PUT_INT(ptr, val) \
   (*(int *) ((char *) (ptr)) = htonl ((int) (val)))
-#define OR_PUT_FLOAT(ptr, val) \
-  (*(UINT32 *) (ptr) = htonf (*(float*) (val)))
-#define OR_PUT_DOUBLE(ptr, val) \
-  (*(UINT64 *) (ptr) = htond (*(double *) (val)))
+inline void
+OR_PUT_FLOAT (char *ptr, float *val)
```

Comment:
please pass by value to keep consistency with the most usual, or_put_int

Reply:
fixed